cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,543 @@
+
+Fields
+++++++
+
+Created
+_______
++--------------+----------------------+
+| Operator     | - =                  |
+|              | - >                  |
+|              | - <                  |
+|              | - BETWEEN            |
+|              | - NOT BETWEEN        |
+|              | - EMPTY              |
++--------------+----------------------+
+| Value        | dates format :       |
+|              | yyyy-mm-dd. If       |
+|              | operator is EMPTY,   |
+|              | values information   |
+|              | are ignored          |
++--------------+----------------------+
+| Example      | .. code-block:: yaml |
+|              |                      |
+|              |   field: created     |
+|              |   operator: =        |
+|              |   value: 2015-01-23  |
++--------------+----------------------+
+
+Updated
+_______
++--------------+----------------------+
+| Operator     | - =                  |
+|              | - >                  |
+|              | - <                  |
+|              | - BETWEEN            |
+|              | - NOT BETWEEN        |
+|              | - EMPTY              |
++--------------+----------------------+
+| Value        | dates format :       |
+|              | yyyy-mm-dd. If       |
+|              | operator is EMPTY,   |
+|              | values information   |
+|              | are ignored          |
+|              |                      |
+|              |                      |
++--------------+----------------------+
+| Example      | .. code-block:: yaml |
+|              |                      |
+|              |   field: updated     |
+|              |   operator: =        |
+|              |   value: 2015-01-23  |
++--------------+----------------------+
+
+Enabled
+_______
++--------------+----------------------+
+| Operator     | - =                  |
++--------------+----------------------+
+| Value        | activated=> true,    |
+|              | deactived => false.  |
++--------------+----------------------+
+| Example      | .. code-block:: yaml |
+|              |                      |
+|              |   field: enabled     |
+|              |   operator: =        |
+|              |   value: false       |
++--------------+----------------------+
+
+Completeness
+____________
++--------------+-----------------------+
+| Operator     | - =                   |
+|              | - >                   |
+|              | - <                   |
++--------------+-----------------------+
+| Value        | Percentage.           |
+|              | /!\ locale and scope  |
+|              | are mandatory         |
++--------------+-----------------------+
+| Example      | .. code-block:: yaml  |
+|              |                       |
+|              |   field: completeness |
+|              |   locale: fr_FR       |
+|              |   scope: print        |
+|              |   operator: =         |
+|              |   value: 100%         |
```

Comment:
value must be into quotation marks "100" and without percentage sign
value: "100"